src/content/docs/apm/agents/python-agent/hosting-services/python-agent-microsoft-azure.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Python agent on Microsoft Azure App Services and Container Apps
+metaDescription: How to install APM for Python on Microsoft Azure App Services and Container Apps.
+redirects:
+  - /docs/agents/python-agent/hosting-services/python-agent-microsoft-azure
+  - /docs/python/python-agent-on-microsoft-azure
+freshnessValidatedDate: never
+---
+
+Learn about special considerations for using Microsoft Azure Container Apps and App Services (formerly known as Web Apps) as a hosting service with New Relic's Python agent.
+
+## What you need [#compatibility]
+
+In addition to the [Compatibility and requirements for the Python agent](/docs/agents/python-agent/getting-started/compatibility-requirements-python-agent), you'll also need the following for this tutorial:
+
+* A workng installation of Python
+* The Windows Azure SDK for Python for your platform
+
+<Callout variant="important">
+  The Python agent cannot capture telemetry for Azure Functions. Instead, you will need to use the [Azure Functions monitoring integration](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list/azure-functions-monitoring-integration/).
+</Callout>
+
+## Installing the Python agent [#install_python]
+
+To add the New Relic Python agent to your Azure application, add `newrelic` to your `requirements.txt` file.
```

Comment:
What do you think about pointing customers to the Python install doc, then prompting them to return to this doc? Are these separate install instructions not present in the Python install doc? @lrafeei